docs/docs/configuration.md

```diff
@@ -49,6 +52,70 @@ The following `Mirror` setting configures BaGetter to index packages from [nuget
 
 :::
 
+### Mirror Authentication
+
+<Tabs>
+  <TabItem value="None" label="None" default>
+    If your mirror does not require authentication, leave the `Mirror` configuration as is and you're good to go.
+  </TabItem>
+
+  <TabItem value="Basic" label="Basic">
+    For basic authentication, set `Type` to `Basic` and provide a `Username` and `Password`:
+
+    ```json
+    {
+        "Mirror": {
+            "Enabled":  true,
+            "PackageSource": "https://api.nuget.org/v3/index.json",
+            "Authentication": {
+                "Type": "Basic",
+                "Username": "username",
+                "Password": "password"
+            }
+        },
+    }
+    ```
+  </TabItem>
+
+  <TabItem value="Token" label="Token (Bearer)">
```

Comment:
I'd prefer "Bearer Token", as it's a known phrase and if someone doesn't know it, putting it into a search engine will provide precise results.